tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/generate_multishop_combination.feature

```diff
@@ -182,3 +182,41 @@ Feature: Generate combination from Back Office (BO) when using multi-shop featur
     And product "product1" should have the following combinations for shops "shop2":
       | combination id | combination name | reference | attributes | impact on price | quantity | is default |
       | product1S      | Size - S         |           | [Size:S]   | 0               | 0        | true       |
+
+  Scenario: Generate combinations in all shops
+    When I generate combinations for product "product1" in all shops using following attributes:
+      | Size  | [S,M]              |
+      | Color | [White,Black,Blue] |
+    Then product "product1" should have the following combinations for shops "shop1,shop2":
+      | id reference   | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SWhite | Size - S, Color - White |           | [Size:S,Color:White] | 0               | 0        | true       |
+      | product1SBlack | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 0        | false      |
+      | product1SBlue  | Size - S, Color - Blue  |           | [Size:S,Color:Blue]  | 0               | 0        | false      |
+      | product1MWhite | Size - M, Color - White |           | [Size:M,Color:White] | 0               | 0        | false      |
+      | product1MBlack | Size - M, Color - Black |           | [Size:M,Color:Black] | 0               | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+    And product "product1" default combination for shop "shop1" should be "product1SWhite"
+    And product "product1" default combination for shop "shop2" should be "product1SWhite"
+
+  Scenario: Generate combinations in all shops when there is already existing combinations in one of shops
```

Comment:
Maybe a third scenario where a combination already exist Then you create for all shops with attributes already in common with the previously create combination to ensure it doesn't create duplicates even with the AllShops constraint